docs/faq.md

```diff
@@ -57,11 +57,31 @@ https://docs.docker.com/desktop/
 
 #### **Q: How can I enable the dashboard for the Traefik ingress controller?**
 
-**A:** The Traefik dashboard is not exposed by default, for security reasons. However, it is possible to expose the dashboard in multiple ways. For instance, using `port-forward` will enable dashboard access:
+**A:** The Traefik dashboard is not exposed by default, for security reasons. However, it is possible to expose the dashboard in multiple ways. This object creation will enable dashboard access:
 
+- `ExposeTraefik.yaml`
 ```
-kubectl port-forward -n kube-system $(kubectl -n kube-system get pods --selector "app.kubernetes.io/name=traefik" --output=name) 9000:9000
```

Comment:
@jparrill , thank you for the PR.

It would be nice to have both ways (kubectl port-forward and helm config) documented here. So, Instead of replacing the `kubectl port-forward..`, Can you please add the HelmChartConfig approach instructions?